marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx

```diff
@@ -227,8 +229,13 @@ const RecordCardView = (props) => {
 
   };
 
-  const submitMergeUri = async (payload) => {
-    await mergeUris(payload);
+  const submitMergeUri = async (uri, payload) => {
+    const documentsHaveMerged = await mergeUris(payload);
+    console.log("documentsHaveMerged", documentsHaveMerged);
```

Comment:
Looks good. Can we clean up the console.logs here? It's probably the culprit for the lint failures as well.